crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -1317,6 +1317,55 @@ module deepbook::clob_v2 {
         }
     }
 
+    /// Clean up expired orders
+    /// Note that this function can reduce gas cost if  orders
+    /// with the same price are grouped together in the vector.
```

Comment:
Can you just expand a little bit on why here? Maybe ` Note that this function can reduce gas cost if orders with the same price are grouped together because we would not need the computation to find the tick_index.`

Reply:
sure!